dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNHvdcLineInterfaceIIDM.cpp

```diff
@@ -248,4 +260,30 @@ HvdcLineInterfaceIIDM::getConverter2() const {
   return conv2_;
 }
 
+boost::optional<double>
+HvdcLineInterfaceIIDM::getDroop() const {
+  return hvdcActivePowerControl_ ? hvdcActivePowerControl_->getDroop() : boost::optional<double>();
+}
+
+boost::optional<double>
+HvdcLineInterfaceIIDM::getP0() const {
+  return hvdcActivePowerControl_ ? hvdcActivePowerControl_->getP0() : boost::optional<double>();
+}
+
+boost::optional<bool>
+HvdcLineInterfaceIIDM::isEnabled() const {
```

Comment:
isEnabled is too generic

Reply:
It is the name of the extension property

Reply:
Please add the extension name for example?

Reply:
OK